docs/core/linux-prerequisites.md

```diff
@@ -127,7 +143,6 @@ To install .NET Core dependencies on RHEL Server:
 > [!NOTE]
```

Comment:
this doesn't need to be in a note. one thing that is not clear for me in the heading title is that you're installing 1.1 here. just noticed when I was in the middle of instructions. shouldn't that be said in the heading or description?